docs/usage/administration/admin_api/registration_tokens.md

```diff
@@ -23,9 +24,6 @@ These objects have the following fields:
   complete a registration.
 - `expiry_time`: The latest time the token is valid. Given as the number of
   milliseconds since 1970-01-01 00:00:00 UTC (the start of the Unix epoch).
-  A more human friendly format will be provided at some point, but in the
-  meantime you can remove the milliseconds and use the `date` command.
-  For example, `date -d '@1625394937'`.
```

Comment:
Oh, I was just commenting on the "A more human friendly format will be provided". I would still keep the `date` example though. That was handy!  :smiley: